doc/installing.md

```diff
@@ -322,6 +322,10 @@ As OpenSUSE stores QEMU firmware files using an unusual filename and location, y
 
 ````
 
+```{note}
+On ARM64 CPUs you need to install AAVMF instead of OVMF for UEFI to work with virtual machines.
+```
+
```

Comment:
I don't think we should need that.

Instead we should just remove all that symlink stuff and make sure that the logic in the code knows to look into /usr/share/qemu too in this case.